cmd/snap-repair/runner.go

```diff
@@ -676,3 +679,35 @@ func (run *Runner) Next(brandID string) (*Repair, error) {
 		}, nil
 	}
 }
+
+// we use designated subroot keys so that we have a subtree of keys and make
```

Comment:
I found it a bit hard to read this sentence.
Perhaps:

_// Limit trust to specific keys while there's no delegation support.
// The obtained assertion stream may also include account keys
// that are directly or indirectly signed by a trusted key._